rockcraft/extensions/gunicorn.py

```diff
@@ -308,7 +308,7 @@ def _wsgi_path_error_messages(self) -> list[str]:
 
         if not has_app:
             return [
-                "flask application can not be imported from app:app in app.py in the project root."
+                "the global variable 'app' was not found in 'app.py' in the project root."
```

Comment:
```suggestion
                "the global variable 'app' was not found in app.py in the project root."
```

Nitpick:

We can usually save some characters by removing the quotation marks from well-known filenames, but we're not always consistent about this.